docs/guides/monitor/anomaly-detection.md

```diff
@@ -0,0 +1,173 @@
+- Familiarity with configuring the Netdata Agent with [`edit-config`](/docs/configure/nodes.md).
+- _Optional_: An Nginx web server running on the same node to follow the example configuration steps.
+
+## Install required Python packages
+
+The anomalies collector uses a few Python packages, available with `pip3`, to run ML training. It requires `numba`,
+`scikit-learn`, `pyod`, in addition to `netdata-pandas`, which is a package built by the Netdata team to pull data from
+a Netdata Agent's API into a Pandas DataFrame. Read more about `netdata-pandas` on its [package
+repo](https://github.com/netdata/netdata-pandas) or in Netdata's [community
+repo](https://github.com/netdata/community/tree/main/netdata-agent-api/netdata-pandas).
+
+```bash
+# Become the netdata user
+sudo su -s /bin/bash netdata
+
+# Install required packages for the netdata user
+pip3 install --user netdata-pandas==0.0.32 numba==0.50.1 scikit-learn==0.23.2 pyod==0.8.3
+```
+
+> If the `pip3` command fails, you need to install it. For example, on an Ubuntu system, use `sudo apt install
+> python3-pip`.
+
+Use `exit` to become your normal user again.
+
+## Enable the anomalies collector
+
+Navigate to your [Netdata config directory](/docs/configure/nodes.md#the-netdata-config-directory) and use `edit-config`
+to open the `python.d.conf` file.
+
+```bash
+sudo ./edit-config python.d.conf
+```
+
+In `python.d.conf` file, search for the `anomalies` line. If the line exists, set the value to `yes`. Add the line
+yourself if it doesn't already exist. Either way, the final result should look like:
+
+```conf
+anomalies: yes
+```
+
+[Restart the Agent](/docs/configure/start-stop-restart.md) with `sudo systemctl restart netdata` to start up the
+anomalies collector. By default, the model training process runs every 30 minutes, and uses the previous 4 hours of
+metrics to establish a baseline for health and performance across the default included charts.
+
+## Configure the anomalies collector
+
+Open `python.d/anomalies.conf` with `edit-conf`.
+
+```bash
+sudo ./edit-config python.d/anomalies.conf
+```
+
+The file contains many user-configurable settings with sane defaults. Here are some important settings that don't
+involve tweaking the behavior of the ML training itself.
+
+- `charts_regex`: Which charts to train against and include in anomaly detection.
+- `charts_to_exclude`: Specific charts, selected by the regex in `charts_regex`, to exclude.
+- `train_every_n`: How often to train the ML models.
+- `train_n_secs`: The number of historical metrics to train ML models on. The default is 4 hours, but if your node
+  doesn't have historical metrics going back that far, consider [changing the metrics retention
+  policy](/docs/store/change-metrics-storage.md).
+- `custom_models`: A way to define custom models that you want anomaly probabilities for, including multi-node or
+  streaming setups.
+
```

Comment:
Could be worth adding a little note in to warn them against doing something silly like trying to set a huge train_n_secs and/or include all charts. That will cause pain. So maybe something like.

> Note that if you set `charts_regex` to cover a lot of charts and/or set `train_n_secs` to be a very large number, then this will impact the resources and time required to train the models. The performance implications here will depend on each node so its best to make incremental changes if you want to include more or different charts and adjust the volume of data used to train the model. `train_max_n` might be useful too here as it will cap via sampling the number of observations actually used to train on.

Reply:
A bit wordy, But just sort of something to stop people blowing up their node is kinda where i'm coming from (without wanting to scare them out of using it either).